salt/modules/saltcheck.py

```diff
@@ -317,6 +317,14 @@
 from salt.utils.json import dumps, loads
 from salt.utils.odict import OrderedDict
 
+try:
+    from junit_xml import TestCase, TestSuite
+
+    HAS_JUNIT = True
+except (ImportError, ModuleNotFoundError):
```

Comment:
Why would you get `ModuleNotFound` and not `ImportError`?

Reply:
Fixed this